commit 62bc8ccf55168bba094db96a07948586ec62af17 (refs/changes/06/4406/3)
Author: Brittany Wald <brittany.wald@joyent.com>
Date:   2018-07-12T19:23:25+00:00 (1 year, 3 months ago)
    
    MANTA-3796 disable directory listing on mako zone nginx
    Reviewed by: Dylan Yep <dylen.yep@joyent.com>
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/sapi_manifests/mako/template b/sapi_manifests/mako/template
index c5775c7..030f7a2 100644
--- a/sapi_manifests/mako/template
+++ b/sapi_manifests/mako/template
@@ -45,7 +45,6 @@ http {
 	send_timeout	300s;
 
 	server {
-		autoindex 	on;
 		listen		80 so_keepalive=10s:30s:10;
 		root		/manta;
 		server_name	localhost;
